### DISCUSSION TITLE
Stream Microsoft Windows application window via Pion

### DISCUSSION THREAD

**Discussion by mahadevans87** [pion/webrtc#1637] [Q&A] [answered]
I would like to understand if we can share an application screen in Windows via webrtc? If so, can someone please share pointers on how to achieve the same?

On Linux, we can make a GStreamer Pipeline grabbing buffers from the X11 Server like so

![image](https://user-images.githubusercontent.com/1255523/104754948-d3c07280-577f-11eb-85bf-fb16bdf9038d.png)

[Reference](https://cloud.google.com/solutions/gpu-accelerated-streaming-using-webrtc#gstreamer_pipeline)

Thanks

**Comment by Sean-Der** [comment, marked as answer]
Hey @mahadevans87 

Check out [webrtc-remote-screen](https://github.com/rviscarra/webrtc-remote-screen) for a real world implementation. You also might be interested in [cloud-morph](https://github.com/giongto35/cloud-morph) it might already be doing what you need! 

We also provide [mediadevices](https://github.com/pion/mediadevices) this implements the `getUserMedia` API and gives you capture+encoding. If it doesn't have the capture types you need it would be an awesome contribution :)

**Reply by mahadevans87** [reply]
Thank you. 

Looks like https://github.com/pion/mediadevices is what I would need. Will try it out and get back. Also would love contribute back to this awesome project.
